server/src/Game/Game.js

```diff
@@ -86,6 +86,12 @@ export default class Game {
         const dt = this._tick() / 1000
         Object.values(this.players).forEach(player => {
             player.socket.emit(EVENTS.SERVER_TICK, dt)
+            const pchange = player.update(dt)
+
+            //Checks for player position changes and updates their position
+            if(pchange){ 
+                player.socket.emit(EVENTS.SERVER_UPDATE_PLAYER, player.getSerialized())
```

Comment:
Could you also call `player.socket.broadcast.emit` on the next line just like you called `player.socket.emit`?
`player.socket.emit` only emits the event to the current player but not all others.
`player.socket.broadcast.emit` emits the event to all other players but not the current player

Hence why we need both called here.